### PR TITLE
feat: Adds CI implementation

### PR DESCRIPTION
**Pull request by andres-rubio-go**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -3,31 +3,31 @@ var path = require("path");
...
   entry: {
     spider: "./src/spider.js",
   },
   devServer: {
     contentBase: __dirname,
   },
   output: {
-    filename: "[name].js",
-    path: __dirname,
+    filename: "spider.js",
```
Comment (jeremytchang) [inline webpack.config.js:14]: `filename: "bundle.js"`

Comment (jeremytchang) [inline webpack.config.js]: `filename: "bundle.js"`

**Review by jeremytchang — CHANGES_REQUESTED** [review]

**Review by jeremytchang — CHANGES_REQUESTED** [review]
Copying the requirements here:

Based off this[ example PR](https://github.com/looker-open-source/viz-radial_gauge-marketplace/pull/24): 
- Copy the .github/workflows directory with its 2 yaml files.
- For README, outside of initial unique info per viz, the rest should be exact copy.
- Update webpack config to output the js bundle to `dist\bundle.js`. This must result from `yarn build` call to ensure release workflow works. Looks like some of the webpack configs are inconsistent, so we’ll need to be careful here and ensure locally the output is correct.
- Remove any minified bundle js files.
- Update gitignore to ignore dist directory
- Make sure the master branch is called `master` for release and ci workflow to work.

**Review by yesudeep — APPROVED** [review]

File: .github/workflows/ci-build.yml [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+on: 
+  push:
+    branches:
+      - master
+  pull_request:
+  workflow_dispatch:
+
+jobs:
+  ci-build-job:
+    uses: looker-open-source/reusable-actions/.github/workflows/marketplace-viz-ci-build.yml@main
```
Comment (yesudeep) [inline .github/workflows/ci-build.yml]: Please add a blank line at the end of this file.

File: .github/workflows/release.yml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+  cancel-in-progress: true
+
+jobs:
+  ci-release-job:
+    permissions:
+      contents: write
+      pull-requests: write
+    uses: looker-open-source/reusable-actions/.github/workflows/marketplace-viz-release.yml@main
+    # Inherits the Looker Automation Bot token to create release PRs and releases
+    secrets: inherit
```
Comment (yesudeep) [inline .github/workflows/release.yml]: Please add a blank line at the end of this file.

**Review by jeremytchang — APPROVED** [review]

**Pull request merged**